.gitlab/dev_container_deploy/docker_linux.yml

```diff
@@ -257,12 +276,39 @@ qa_agent:
   needs:
     - docker_build_agent7
     - docker_build_agent7_arm64
+    - docker_build_ot_agent7
+    - docker_build_ot_agent7_arm64
     - docker_build_agent7_windows1809
     - docker_build_agent7_windows2022
   variables:
     IMG_REGISTRIES: agent-qa
-    IMG_SOURCES: ${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-amd64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-arm64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-win1809-amd64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-winltsc2022-amd64
-    IMG_DESTINATIONS: agent:${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}
+  parallel:
+    matrix:
```

Comment:
@KevinFairise2 I did this because I was getting a line too long error in the yaml validation. I was hoping the multiplatform magic would work in a job matrix, but I see it doesn't. I've split the QA publishing of `ot` images and the regular images into two different jobs, and that appears to work. Please let me know if this looks good to you now.

Reply:
Should be fine like that! Thanks for the change